MVC_Activity01/MVC_Activity01/App_Start/RouteConfig.cs

```diff
@@ -16,7 +16,7 @@ public static void RegisterRoutes(RouteCollection routes)
             routes.MapRoute(
                 name: "Default",
                 url: "{controller}/{action}/{id}",
-                defaults: new { controller = "Home", action = "Index", id = UrlParameter.Optional }
+                defaults: new { controller = "Home", action = "About", id = UrlParameter.Optional }
```

Comment:
can you explain why you modify this?

Reply:
It change Index to About sir so that when I will run the program it will show the about page.

Reply:
You don't need to do that, and in the task description it says `Clear the home page...`
Also, this change has introduce a bug when you click Back in the create/edit customer page

Reply:
ok sir rey @rjs123431 noted sir thank you for the info sir.